### PR TITLE
Booking initial page

### PR DESCRIPTION
**Pull request by jakobpalmborg**
I have created the initial page for booking. 

I added a function for rendering the top of the booking page so that you can see salon, title and time for the booking you have clicked.

For now I have some placeholder data and I have prepaired for it to work with an eventlistener for the booking-button later.

### CODE REVIEW

File: booking.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Booking</title>
+    <link rel="stylesheet" href="static/css/main.css" />
+    <script type="module" src="src\JS\booking.js" defer></script>
+  </head>
+  <body>
+    <header class="sec">
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
```
Comment (mkor92) [inline booking.html:13]: Would be great if you could change " L " in logo.png to uppercase. Think thats what creating problems when trying to load logo on github pages

File: booking.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Booking</title>
+    <link rel="stylesheet" href="static/css/main.css" />
+    <script type="module" src="src\JS\booking.js" defer></script>
```
Comment (mkor92) [inline booking.html:8]: Maybe link to main.js, and then import needed functions from booking.js.

**Review by mkor92 — APPROVED** [review]
Great work. I added som comments, minor changes if any.

File: booking.html (hunk 1 above)
Comment (jakobpalmborg) [reply booking.html:13]: Or change the image to lowercase? what do you think?

**Review by DanneCarlson — APPROVED** [review]
Hi! Nice work Jakob. I didn't find anything (read Markus comments and guess that you've concidered those small issues)

File: booking.html (hunk 1 above)
Comment (mkor92) [reply booking.html:13]: That works to!

**Pull request merged**